client/jetpack-connect/index.js

```diff
@@ -46,6 +50,12 @@ export default function() {
 			page.redirect( `/jetpack/connect/store${ params.interval ? '/' + params.interval : '' }` )
 	);
 
+	if ( isLoggedOut ) {
+		page( '/jetpack/connect/plans/:interval(yearly|monthly)?/:site', ( { path } ) =>
+			page.redirect( `/log-in?redirect_to=${ path }` )
```

Comment:
@tug is this the right approach (and the correct login link) for prompting a log-in on a particular route?

Reply:
Almost :)
We usually use the `login` route helper:
```js
import { login } from 'lib/paths';
...
page.redirect( login( { isNative: true, redirectTo: path } ) );
```

Reply:
Thank you!